src/commands/attest/apply.ts

```diff
@@ -8,6 +8,12 @@ import path from 'path'
 import {convertFullPathToFilename} from '../../utils/global'
 
 export default class ApplyAttestation extends Command {
+    static usage = 'attest apply -i <input-hdf-json>... <attestation>... -o <output-hdf-path>'
+
+    static description = 'Apply one or more attestation files to one or more HDF results sets'
+
+    static examples = ['saf attest apply -i hdf1.json hdf2.json attestation.json -o outputDir']
```

Comment:
Can we add an example for outputting to a instead of a folder? Like this:
`saf attest apply -i hdf_results.json attestation_results.json -o hdf_with_attestation_results.json`

Reply:
done